opm/core/wells/DynamicListEconLimited.hpp

```diff
@@ -80,6 +80,22 @@ namespace Opm
             }
         }
 
+        void removeShutWell(const std::string& well_name) {
```

Comment:
I could not find these three functions are used anywhere. If they are not used, maybe they should be removed.

Reply:
We no longer use DynamicListEconLimited.hpp in Flow. Is this used anywhere els? If not I can remove it completely.

Reply:
Maybe flow_legacy is still using it?